ketcher-autotests/tests/User-Interface/Zoom-Changes/zoom-changes.spec.ts

```diff
@@ -44,4 +59,100 @@ test.describe('Zoom changes', () => {
 
     expect(zoomInputValue).toBe('80%');
   });
+
+  test('Zoom In button verification', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1761
+    */
+    await pressButton(page, STRUCTURE_LIBRARY_BUTTON_NAME);
+    await page.getByRole('tab', { name: 'Functional Groups' }).click();
+    await selectFunctionalGroups(FunctionalGroups.CO2Et, page);
```

Comment:
I propose to use some other template or may be expand this group. That's because it is not really clear on screenshot, whether the structure was just shifted or zoomed.

Reply:
expanded

Reply:
Done